ociinstaller/mediatypes.go

```diff
@@ -59,7 +60,7 @@ func MediaTypeForPlatform(imageType ImageType) []string {
 	case ImageTypeDatabase:
 		return []string{fmt.Sprintf(layerFmtTar, imageType, runtime.GOOS, arch)}
 	case ImageTypeFdw:
-		return []string{fmt.Sprintf(layerFmtGzip, imageType, runtime.GOOS, arch)}
+		return []string{fmt.Sprintf(layerFmtGzip, imageType, runtime.GOOS, utils.UnderlyingArch())} // Detect the underlying architecture(amd64/arm64)
```

Comment:
```
// detect the underlying architecture(amd64/arm64)
// we have to do this rather than just using runtime.GOARCH  because......
arch := utils.UnderlyingArch()
return []string{fmt.Sprintf(layerFmtGzip, imageType, runtime.GOOS, arch)} 
```